lib/models/ios_config.dart

```diff
@@ -23,3 +27,12 @@ enum AuthMiddleware {
   /// WebView will be used.
   webView
 }
+
+/// Types of tenant to authenticate against.
+enum TenantType {
```

Comment:
Can't there be more than 2 values?
referring here: https://learn.microsoft.com/en-us/entra/msal/objc/install-and-configure-msal#configuring-your-project-to-use-msal

Reply:
I assume you're referring to this chart? https://learn.microsoft.com/en-us/entra/identity-platform/msal-configuration#map-microsoft-entra-authority--audience-to-microsoft-identity-platform-endpoints

Interestingly I can only see this being refered to as being required on Android, despite that fact that I'm able to login to my B2C tenant on Android without any changes to your code - very odd!

From looking at that chart, it seems to suggest that all non-B2C tenants use the same authority so, as my code change is to deal with this issue, we should only need to make a distinction between B2C and non-B2C tenants. I could potentially rename the first enum value to something other than 'entraIDAndMicrosoftAccount', though I went for that as it mostly matches the existing naming conventions. I'm happy to hear your suggestions on this.

The other slight issue I have, and the main reason why I've tried to avoid changing your existing code, is that I can only really test against my B2C tenant and using these changes I'm able to login against my B2C client on both Android and iOS. If there are different settings required for "AzureADMyOrg" or "AzureADMultipleOrgs", I'm not able to test this I'm afraid.

Reply:
No problem, Android is working without changing the code because we are giving the `JSON` file with all the configuration. no extra work is needed. for the iOS, we need to give all the configuration by dart to swift code.
Anyway, merging it now.